README.md

```diff
@@ -1,180 +1,35 @@
-# OCDS Extension Template Repository
+# Withheld information
 
-[OCDS extensions](https://standard.open-contracting.org/latest/en/extensions/) can be used to declare additional fields, objects and sections that will be included in an OCDS file, but which are not in the core schema, and to add or extend codelists. When any OCDS publication cites an extension in its' package metadata, it should be validated against the schema with that extension applied. 
+Adds an array of `withheldInformation` objects to describe data points that are embargoed or permanently withheld from publication.
```

Comment:
I think that an embargo can be permanent so I'm not sure it's quite the right word.

```suggestion
Adds an array of `withheldInformationItem` objects to describe items of information whose publication is temporarily or permanently withheld.
```